### ISSUE TITLE
JBR Linux distribution: cannot launch

### ISSUE THREAD

**Issue by iromeo** [JetBrains-Research/jbr#14]
JBR Linux distribution: cannot launch:

1) jbr.sh isn't executable
2) bundled java.sh not executable => JBR fails to launch it and look for any other JVM on machine

**Comment by iromeo** [comment]
Workaround
```
cd jbr_distr
chmod u+x ./bins/jbr.sh
chmod u+x ./jre64/bin/java.sh
```

**Comment by iromeo** [comment]
Fixed JBR 1.0 build #4789
